plugins/user-authenticators/ldap/src/main/java/org/apache/cloudstack/ldap/LdapManagerImpl.java

```diff
@@ -65,6 +66,7 @@
 @Component
 public class LdapManagerImpl extends ComponentLifecycleBase implements LdapManager, LdapValidator {
     private static final Logger LOGGER = Logger.getLogger(LdapManagerImpl.class.getName());
+    public static final String REMOVING_LDAP_LINK_ON_DOMAIN = "Removing link between LDAP: %s - type: %s  and account: %s on domain: %s";
```

Comment:
I think its better to define a method that generates the resulting string message with the parameters already set instead of this

Reply:
it is used in two places. I don't understand what you mean would be better.